src/server/ua_subscription_events.c

```diff
@@ -425,17 +459,22 @@ UA_Event_addEventToMonitoredItem(UA_Server *server, const UA_NodeId *event,
 }
 
 static const UA_NodeId objectsFolderId = {0, UA_NODEIDTYPE_NUMERIC, {UA_NS0ID_OBJECTSFOLDER}};
-static const UA_NodeId parentReferences_events[2] =
+static const UA_NodeId parentReferences_events[4] =
```

Comment:
why not let the compiler deduce the array size? if the UE_ENABLE_SUBSCRIPTIONS_ALARMS_CONDITIONS is off, there will be two unused and uninitialized entries

Reply:
The tinyCC compiler does not like the array length being left out...
Probably travis will fail.

Reply:
> 
> 
> The tinyCC compiler does not like the array length being left out...
> Probably travis will fail.

that's right, travis fails when array length is being left empty. I would put the array defintion in ifdef block, so we won't have wrong array length when A&C are disabled.